bindings/node.js/test.ts

```diff
@@ -47,6 +50,12 @@ describe("C-KZG", () => {
     );
   });
 
+  it("verifies the aggregate proof of empty blobs and commitments", () => {
+    expect(verifyAggregateKzgProof([], [], computeAggregateKzgProof([]))).toBe(
+      true,
+    );
+  });
```

Comment:
Currently throws `verify_aggregate_kzg_proof failed with error code: 1` which means `verify_aggregate_kzg_proof` returned `C_KZG_BADARGS`

Reply:
I think this is fine while https://github.com/ethereum/consensus-specs/pull/3093 has not been merged